src/chttpd/src/chttpd_db.erl

```diff
@@ -1637,15 +1637,15 @@ db_attachment_req(#httpd{method = 'GET', mochi_req = MochiReq} = Req, Db, DocId,
         atts = Atts
     } = Doc = couch_doc_open(Db, DocId, Rev, Options),
     Att = get_existing_attachment(Atts, FileName),
-    [Type, Enc, DiskLen, AttLen, Md5] = couch_att:fetch(
-        [type, encoding, disk_len, att_len, md5], Att
+    [Type, Enc, DiskLen, AttLen, Digest] = couch_att:fetch(
+        [type, encoding, disk_len, att_len, digest], Att
     ),
     Refs = monitor_attachments(Att),
     try
         Etag =
-            case Md5 of
+            case Digest of
                 <<>> -> chttpd:doc_etag(Doc);
-                _ -> "\"" ++ ?b2l(base64:encode(Md5)) ++ "\""
+                _ -> "\"" ++ ?b2l(base64:encode(Digest)) ++ "\""
```

Comment:
Here we can probably keep the Digest variable as we're passing along whatever we got from the attachment into the ETag.